dashboard/app/models/code_review_group_member.rb

```diff
@@ -13,4 +13,9 @@
 #  index_code_review_group_members_on_follower_id           (follower_id)
 #
 class CodeReviewGroupMember < ApplicationRecord
+  belongs_to :follower
+
+  def name
```

Comment:
I think ideally we have as little logic as possible in here -- this is just really for readability so doesn't seems fine to me, but maybe could you add a note that suggests that new logic should either live in `Followers` or `CodeReviewGroups`, something like [this](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/levels_script_level.rb#L16)?

The pattern I was going for is something like this:

https://guides.rubyonrails.org/association_basics.html#creating-join-tables-for-has-and-belongs-to-many-associations

Although this isn't quite the same, since each follower only belongs to one group, but I think it's the same idea?

Reply:
I'll add a note to keep the logic minimal